### DISCUSSION TITLE
New Player and On Boarding - In Game Actions for your Bumpkin Bunch

### DISCUSSION THREAD

**Discussion by CryptoMeatBoy** [sunflower-land/sunflower-land#2352] [Ideas]
**Problem:**   
I have friends that enjoy this game but are constantly asking me for resources.   I have these resources but the trading market forces me to set price limits because if I set them low to be available then they get sniped and I am completely lost on my resources and some stranger just got gains.  New players do not understand why I cannot just help them easily.

**Suggested Solution:**  

First most this only will function for verified farms on either side of the party.  There will be no limit on level, expansion or otherwise.
You will be allowed to have dedicated friends which will become your Bumpkin Bunch.
These friends and only these friends and you will be able to send and receive a limited amount of recourses between.  
CAP Suggestions:   5000 SF, 2500 Potato, 1000 Pumpkins, 500 Carrots, 500 Wood, 200 Stone, 100 Iron, 25 Gold..  etc so on and please adjust to community agreement.
Two friends do not share friends.  If I have 5 friends and you are one of them, you may have 4 other different friends.  I would be the 5th friend in your bunch.

**What this will yield:**

New players which are brought on by word of mouth will be enhanced and no so easily dissuaded while initially building their first farm.  They will feel that they have a real companion which can assist and guide them.    There are people that have a cell phone who do not have a access to add funds how some of us have acquired.    This will promote on boarding as a whole.

**POSSIBILITIES:**

This really could be made into a larger function with limitless possibilities if implemented properly.
One thought is allowing people in your bunch to tend your farm
Another would be Bumpkin Bunch Brawls
Maybe 2 people will actually meet in this game and make a bunch to continue to have a real life together.
limitless possibilities. 

**Preventing Exploitation:**

Any farm must be verified.
You may only have 5 farms which are in your Bumpkin Bunch for trading resources.
Limits on what you may trade between farms.
Friends must be registered with each other.
POSSIBILE OTHER TOPIC IF NOT ALREADY IN PLACE:   Mechanic which watches farms in that bunch and how/when/where they operate.   For example..  If 3 farms are operating from the same IP and actions are taking place at the same exact time or one after another..   this is just a thought.  I trust you all know what I mean and have alternative solutions.
Please add suggestions here.

**Comment by VZozul** [comment]
The idea may be good, but alas, it will just be a transfer of resources between their own farms. Verification will not help here, you yourself know that you can buy verification or ask friends who do not play to go through verification. In many verifications, I scan a photo of a person or an edited document, just print a photo, even imitate head turns. You can buy a couple of inexpensive mobile phones, connect a cellular connection and that's it. And this cannot be overcome, this is the problem of cryptocurrency, which, unfortunately, is rotten by greed and the desire for easy money. So if you want to help someone, pay 30% to withdraw resources.. it's not that much, but it will support the economy of the game you play, which you love and of course you want the game to continue to live.

**Reply by rasheed19871** [reply]
This is a danger suggestions people will start five five accounts and transferring resources without worrying
